main.py

```diff
@@ -2,61 +2,98 @@
 from services import Services
 from application import App
 from PyQt5.QtWidgets import QApplication
-
+import logging
+from datetime import date, timedelta
+import screen_brightness_control as sbc
+import darkdetect
 
 BASE_URL = 'https://api.olhar.media/'
 
 service = Services()
 
-logger = service.get_logger('main')
+current_city = service.get_current_city()
 
-def main():
-    # try:
-    #     logger.info('Fetching script...')
-    #     script = service.fetch_script(f'{BASE_URL}?getconfigupdate&equipid=1')
-    #     if script:
-    #         if service.is_valid_bash(script):
-    #             with open('script.sh', 'wb') as f:
-    #                 f.write(script.encode())
-    #             service.run_bash('./script.sh')
-    #             os.remove('./script.sh')
-    #             logger.info('Script executed successfully')
-    #         elif service.is_valid_python(script):
-    #             with open('script.py', 'wb') as f:
-    #                 f.write(script.encode())
-    #             service.run_python('./script.py')
-    #             os.remove('./script.py')
-    #             logger.info('Script executed successfully')
-    # except Exception as e:
-    #     logger.error(e)
 
+logging.basicConfig(filename=f'./assets/logs/{date.today()}.log', level=logging.DEBUG)
+logger = logging.getLogger('main')
+
+logger.setLevel(logging.DEBUG)
+
+def set_brightness():
+    if darkdetect.isDark():
+        sbc.fade_brightness(50)
+    else:
+        sbc.fade_brightness(100, increment = 10)
+    
+
+def delete_old_logs():
+    directory = './assets/logs'
```

Comment:
Make it a constant and share in all places using it.

Reply:
✅